OutOfSchool/OutOfSchool.WebApi/Services/WorkshopServicesCombiner.cs

```diff
@@ -228,10 +228,9 @@ public async Task<List<ShortEntityDto>> GetWorkshopListByProviderAdminId(string
     }
 
     /// <inheritdoc/>
-    public async Task<SearchResult<T>> GetByProviderId<T>(Guid id, ExcludeIdFilter filter)
-        where T : WorkshopBaseCard
+    public async Task<SearchResult<WorkshopProviderViewCard>> GetByProviderId(Guid id, ExcludeIdFilter filter)
```

Comment:
(Code style) 5 lines of method body could be rewritten nto 1 line, with removing excessive `async`/`await` keywords and `ConfigureAwait` that (I hope) resides on controller level
```
public Task<SearchResult<WorkshopProviderViewCard>> GetByProviderId(Guid id, ExcludeIdFilter filter)
    => workshopService.GetByProviderId(id, filter);
```

Reply:
It resides on controller level, so I removed them. Thank you!